test/e2e/support/utils/set-viewport.js

```diff
@@ -3,6 +3,11 @@
  */
 import { waitForPageDimensions } from './wait-for-page-dimensions';
 
+/**
+ * Set browser viewport to specified type.
```

Comment:
Can we use the form of 3rd person for consistency?

Reply:
Sure, updated.